package.json

```diff
@@ -4,7 +4,8 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "test": "ava"
+    "test": "ava",
+    "publish": "npm publish"
```

Comment:
not important, but it's not good practice to have a "publish" script. If there are prepublish actions, they'll be executed automatically if a script called "prepublish" is created

Reply:
[Removed](https://github.com/UniversalDataTool/udt-labelme-converter/pull/2/commits/793c5b5ca975ddad4c46a546ea43ea236ccb78ab)